prusa/link/cameras/picamera_driver.py

```diff
@@ -0,0 +1,311 @@
+"""Contains implementation of a driver for Rpi Cameras"""
+import gc
+import logging
+import selectors
+
+from typing import Dict, Optional, Callable
+
+from prusa.connect.printer.camera import Resolution
+from prusa.connect.printer.camera_driver import CameraDriver
+from prusa.connect.printer.const import CapabilityType, NotSupported
+
+from . import v4l2
+from .encoders import MJPEGEncoder, BufferDetails, \
+    get_appropriate_encoder
+from ..printer_adapter.updatable import prctl_name
+from ..util import is_potato_cpu
+
+log = logging.getLogger(__name__)
+
+try:
+    PICAMERA_SUPPORTED = True
```

Comment:
Another solution is set `PICAMERA_SUPPORTED` to False first, and in try block after import from libcamera, set it to True.

Reply:
i had it that way, i liked the symmetry of this one

Reply:
Now, just because you mentioned it, I'll use an else block which you don't like but is perfectly valid. Muhehe :D